### PR TITLE
add CI options and utils

### PR DESCRIPTION
**Pull request by shunjizhan**
similar to our previous waffle examples https://github.com/AcalaNetwork/bodhi.js/tree/master/examples/waffle, we need to add ability to switch between using local (default) url or CI url.

This commit copied the helper util functions from waffle examples to changed waffle tests accordingly.

### CODE REVIEW

File: utils.ts
Comment (shunjizhan) [inline utils.ts]: this method is not used now, but is used by old waffle examples, so might be useful in the future for new waffle tutorials

**Comment by ThunderDeliverer** [comment]
This is great! 👌  Before we merge this PR, I need to document the changes and update the tutorials, so don't merge it just yet.

**Comment by shunjizhan** [comment]
I think we are ready to merge this? It's utils can be reused by #3

**Comment by ThunderDeliverer** [comment]
Let me document it today, I'll merge it and add the helper to the other tutorials

**Comment by ThunderDeliverer** [comment]
@shunjizhan why don't we use the https://github.com/AcalaNetwork/waffle-tutorials/blob/6928e9b885febd073879e1ed6b8b7e6a339da126/hello-world/src/setup.ts as it already supports the `URL` and `SEED` override?

**Comment by shunjizhan** [comment]
yeah we actually could, `setup` is like a superset of `getTestProvider`,  I didn't choose to separate them, just followed what the original waffle examples did, and factored them out.

Do you prefer we use setup for all? I am fine with either way

**Comment by ThunderDeliverer** [comment]
Yeah, lets only have one way of setting up the `provider` and `wallets`. I'll move the `setup.ts` to `utils` folder and rewrite the relevant parts of the tutorials.

**Comment by ThunderDeliverer** [comment]
@shunjizhan I migrated general utility file into separate utility folders within each tutorial. Take a look and if you have no comments, we can merge this. :raised_hands:

File: utils.ts
Comment (shunjizhan) [inline utils.ts]: so we don't need this file anymore, and can remove it?

Comment (ThunderDeliverer) [reply utils.ts]: I was certain that I pushed the commit that removed this file. It's up now :see_no_evil:

**Review by ThunderDeliverer — APPROVED** [review]
Let's do this! :rocket:

**Pull request merged**
